libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKUITableViewCell.m

```diff
@@ -51,7 +51,7 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
     UIImage *image = [SFSDKUITableViewCell resizeImage:pimage  size:CGSizeMake(kImageWidth, kImageHeight)];
     
     self.layer.borderWidth = kHorizontalSpace/2;
-    self.layer.borderColor = [UIColor salesforceSystemBackgroundColor].CGColor;
+    [self updateLayerColor];
```

Comment:
Pulled this out because CGColors don't automatically update on appearance change